crates/astria-sequencer/src/service/mod.rs

```diff
@@ -18,6 +18,7 @@ impl AbciCode {
     pub(crate) const UNKNOWN_PATH: Self = Self(1);
     pub(crate) const INVALID_PARAMETER: Self = Self(2);
     pub(crate) const INTERNAL_ERROR: Self = Self(3);
+    pub(crate) const INVALID_NONCE: Self = Self(4);
```

Comment:
Should these be defined outside the sequencer crate for use in other binaries (ie composer presumably needs to know what the code means?)

Reply:
see #369